Manito/Manito/Screens/CheckRoom/UIComponent/PeopleInfoView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PeopleInfoView.swift
+//  Manito
+//
+//  Created by COBY_PRO on 2022/06/16.
+//
+
+import UIKit
+
+import SnapKit
+
+final class PeopleInfoView: UIView {
+    
+    // MARK: - Property
+    
+    private var peopleImageView: UIImageView = {
```

Comment:
p5;

```suggestion
    private var peopleImageView = UIImageView(image: ImageLiterals.imgNi)
```
이렇게 해도 됩니다!

Reply:
[2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!